.github/workflows/build_test_release.yaml

```diff
@@ -0,0 +1,142 @@
+          conda clean -ya;
+        else
+          conda deactivate
+          pip install virtualenv
+          git clone https://github.com/pyenv/pyenv.git ~/.pyenv
+          export CFLAGS="-fPIC -g"
+          ~/.pyenv/bin/pyenv install --force ${python_version}
+          virtualenv -p ~/.pyenv/versions/$(~/.pyenv/bin/pyenv latest ${python_version})/bin/python3 ~/venvs/multipy
+          source ~/venvs/multipy/bin/activate
+          pip install \
+          torch torchvision torchaudio \
+          --extra-index-url https://download.pytorch.org/whl/cpu;
+        fi
+        echo "::endgroup::"
+        
+        echo "::group::Install"
+        python -m pip install -e . --install-option="--cudatests"
+        echo "::endgroup::"
+        
+        echo "::group::Generate examples"
+        python multipy/runtime/example/generate_examples.py
+        cd examples
+        cmake -S . -B build/ -DMULTIPY_PATH=".." && cmake --build build/ --config Release -j
+        cd -
+        echo "::endgroup::"
+        
+        export PYTHONPATH=$(pwd)
+        export LIBTEST_DEPLOY_LIB=$(pwd)/multipy/runtime/build/libtest_deploy_lib.so
+        export LD_LIBRARY_PATH=/opt/conda/lib/:$LD_LIBRARY_PATH
+        
+        echo "::group::Test C++"
+        multipy/runtime/build/test_deploy
+        echo "::endgroup::"
+        
+        echo "::group::Test Pybind"
+        python multipy/runtime/test_pybind.py
+        echo "::endgroup::"
+        
+        echo "::group::Run examples"
+        examples/build/hello_world_example
+        python3 examples/quickstart/gen_package.py
+        ./examples/build/quickstart my_package.pt
+        ./examples/build/movable_example
+        echo "::endgroup::"
+        
+        echo "::group::Benchmark"
+        ./multipy/runtime/build/deploy_benchmark 2 none jit multipy/runtime/example/generated/resnet
+        echo "::endgroup::"
+        
+        if [[ ${{ inputs.python3-minor-version }} -gt 7 ]];
+        then 
+          echo "::group::Compat test"
+          pip install -r compat-requirements.txt
+          multipy/runtime/build/interactive_embedded_interpreter --pyscript multipy/runtime/test_compat.py
+          echo "::endgroup::";
+        fi
+        
+        echo "::group::Test GPU"
+        # Separating GPU tests due to issues with py37 and py38: https://github.com/pytorch/multipy/issues/239
+        pip install --upgrade --force-reinstall torch \
+        --extra-index-url https://download.pytorch.org/whl/cu116
+        multipy/runtime/build/test_deploy_gpu
+        echo "::endgroup::"
+        
+        echo "::group::Create tarball"
+        cp -r multipy/runtime/build/dist/* .
+        rm -rf multipy/runtime/build
+        tar -czf multipy_runtime_python${python_version}.tar.gz multipy/
+        echo "::endgroup::"
+        
+        echo "::group::Move artifact for upload"
+        mv multipy_runtime_python${python_version}.tar.gz ${RUNNER_ARTIFACT_DIR}/
```

Comment:
@seemethere had some comments on your test-infra PR as a way to improve this. Happy to approve after you fix that.

Reply:
Adopted (even pushed that commit shortly before you commented) and can confirm that moving the artifact directly to that folder works excellent:

<img width="1682" alt="image" src="https://user-images.githubusercontent.com/19345882/207973405-d27fd4a7-ffee-40de-8652-15b1e9ba353f.png">